### DISCUSSION TITLE
Migrating from 4x - 5x: Dynamic Recipient field doesn't have config / source carried over.

### DISCUSSION THREAD

**Discussion by alexmillman23** [solspace/craft-freeform#1566] [Feature Requests]
### What happened?

We have a dynamic_recipients field that is pointed toward an Entries source. When performing the migration from 4.1.x to 5.1.0, that config/source is not migrated over and the Dropdown field that it turns into has no options.

This also affects existing submissions since the previously filled out field no longer has values. So they're gone.

### How can we reproduce this?

1. On version 4.1.x, create a dynamic_recipients field type and assign to a form.
2. Set the source to an entries or similar
3. Update to 5.1.x and review the newly created Dropdown version of it not have its settings carried over

### Freeform Edition

Pro

### Freeform Version

5.1.3

### Craft Version

4.1.8

### When did this issue start?

After upgrading from older Freeform version

### Previous Freeform Version

4.1

**Comment by kjmartens** [comment]
Sorry for the delay and the trouble you experienced @alexmillman23.

I will have a developer check into this shortly. 🙂

**Comment by kjmartens** [comment]
Unfortunately this can get fairly tricky for us to account for in the migration, so it's now on our list for future consideration. 🙂

**Comment by thisisjamessmith** [comment]
Just to add to this issue, it also no longer appears to be possible in FF5 to sort the source options (entries in my case) by "Structure" which is a necessary requirement for one of our clients.

**Reply by kjmartens** [reply]
Sorry about this @thisisjamessmith.

I have logged the "Orderby Structure" issue and we'll see if we can get that added shortly. 🙂

**Reply by kjmartens** [reply]
@thisisjamessmith We've added support back for this in 5.9.14+ 🙂
